.env.example

```diff
@@ -5,3 +5,8 @@ AA_TOKEN="YOUR_TOKEN_HERE"
 AA_INFERENCE_ENDPOINT="AA API Endpoint Here"
 HF_TOKEN="YOUR_TOKEN_HERE"
 OPENAI_API_KEY="YOUR_TOKEN_HERE"
+# Optional Weights & Biases configuration
+WANDB_API_KEY="YOUR WANDB API KEY HERE"
+# Additional wandb environment variables can be set here or passed as kwargs in the cli
+WANDB_PROJECT="YOUR WANDB PROJECT NAME CAN OPTIONALLY BE SET HERE"
```

Comment:
Why `WANDB_PROJECT` and `WANDB_ENTITY` are here if there are CLI arguments at the same time?